helm/README.md

```diff
@@ -164,7 +164,8 @@ The deployment is configured via values.yaml file.
 | maxCPU | Maximum CPUs to allocate to worker pod | 2 |
 | maxBlocks | Maximum number of worker pods to spawn | 100 |
 | maxWorkersPerPod | How many workers will be scheduled in each pod | 1 |
-| endpointUUID   | (Required) Specify a UUID for this endpoint. | |
+| endpointName | (Optional) Specify a name for registration with the funcX web services. | The release name (Release.Name) |
+| endpointUUID | (Required) Specify a UUID for this endpoint. | |
```

Comment:
Some lines have '.' at the end others don't.  Not this PR but we prob should be consistent.

Reply:
Ooh; but I'm already modifying at least those two lines.  I'll go ahead and removes those periods.  Both of those were my fault, I think, anyway.  Good eye.